src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/prisonperson/dpsmodel/PhysicalAttributesDto.kt

```diff
@@ -0,0 +1,58 @@
+/**
+ *
+ * Please note:
+ * This class was auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ *
+ */
+
+@file:Suppress(
+  "ArrayInDataClass",
+  "EnumEntryName",
+  "RemoveRedundantQualifierName",
+  "UnusedImport",
+)
+
+package uk.gov.justice.digital.hmpps.prisonertonomisupdate.prisonperson.dpsmodel
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * Physical Attributes
+ *
+ * @param height
+ * @param weight
+ * @param hair
+ * @param facialHair
+ * @param face
+ * @param build
+ * @param leftEyeColour
+ * @param rightEyeColour
+ */
+
+data class PhysicalAttributesDto(
```

Comment:
The OpenAPI generator generated some classes with compile errors so I couldn't use it. Therefore I just copied the generated classes I need and removed all the OpenAPI generation for the prison person API. (I did try using asking the OpenAPI generator to only generate the classes I need but couldn't get it working).